### PR TITLE
Feature/stp/actions/reducers

### PR DESCRIPTION
**Pull request by Anthony-Cortese**
### Description

**Invited Collaborator(s) to Review
Jaden, Kazden, John

**What work was done?**
Added the actions and reducers for the service type programs.
**Why was this work done?**
This was created in order to have a relationship between the service types and the programs. Each program has a specific set of service types, and we need those relationships for the form to be functional and.

**What feature / user story is it for?**
https://trello.com/b/U8UUCBeb/family-promise-service-tracker

### Type of change
- New feature (non-breaking change which adds functionality)
- Actions and reducers were created.

Change Status
- Completed, ready to review and merge

Loom Recording:
https://www.loom.com/share/c052ad9a939040f8a2371326f9d09f3b

### Checklist
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- My code has been reviewed by at least one peer
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- There are no merge conflicts

### CODE REVIEW

**Comment by john-laubscher** [comment]
Nice naming conventions! Nice job pushing out that much code

**Review by john-laubscher — APPROVED** [review]

File: src/state/index.js [hunk 1]
```diff
@@ -29,6 +30,7 @@ export const rootReducer = combineReducers({
   location: locationReducer,
   serviceRating: serviceRatingReducer,
   gender: genderReducer,
+  serviceTypeProgramsReducer: serviceTypeProgramsReducer,
```
Comment (KazdenJolley) [inline src/state/index.js:33]: I would rename this to 'serviceType'

Comment (Anthony-Cortese) [reply src/state/index.js:33]: Oh I see, get rid of the reducer on the key.

**Review by KazdenJolley — CHANGES_REQUESTED** [review]
I think you should update the key in the src/state/index.js file to 'serviceTypeProgram' to follow the naming convention used for the other elements in the store.

**Review by KazdenJolley — APPROVED** [review]
Looks good! Good job catching that uppercase letter in the file name!

**Pull request merged**